docs/send-data/sumo-logic-opentelemetry/release.md

```diff
@@ -0,0 +1,270 @@
+---
```

Comment:
I would leave this file in the OT Distro repository and not move it into help.sumologic.com